packages/design-systems/panda.config.ts

```diff
@@ -1,12 +1,18 @@
 import { defineConfig } from "@pandacss/dev";
+import { globalCss } from "../design-systems/src/theme/global-css";
+import { conditions } from "../design-systems/src/theme/conditions";
 import { recipes } from "./src/theme/recipes";
 import { semanticTokens } from "./src/theme/semantic-tokens";
 import { slotRecipes } from "./src/theme/slot-recipes";
 import { textStyles } from "./src/theme/text-styles";
 import { tokens } from "./src/theme/tokens";
 
 export default defineConfig({
+  emitPackage: true,
   include: ["./src/**/*.{js,jsx,ts,tsx}"],
+  jsxFramework: "react",
+  outdir: "../../../node_modules/@tailor-platform/styled-system",
```

Comment:
design-systemsとstorybookで暗黙的に同じstyled-systemsを使わないようにしたいので、outdirでルートにstyled-systemsを吐かないようにしたいです。ここで邪魔になる `packages/datagrid` はこの気に削除してしまってOKです。

```suggestion
  outdir: "@tailor-platform/styled-system",
```

その上で、storybookからはpackagesとしてちゃんと扱ってbuildinfoを参照するようにしたいです。

```diff
diff --git a/apps/storybook/panda.config.ts b/apps/storybook/panda.config.ts
index a21d80e..d861f43 100644
--- a/apps/storybook/panda.config.ts
+++ b/apps/storybook/panda.config.ts
@@ -5,8 +5,9 @@ export default buildPandaConfig(
   defineConfig({
     include: [
       "./src/stories/**/*.{js,jsx,ts,tsx}",
-      "./../../packages/design-systems/src/**/*.{js,jsx,ts,tsx}",
       "./node_modules/@tailor-platform/design-systems/dist/panda.buildinfo.json",
     ],
+    importMap: "@tailor-platform/design-system",
+    outdir: "@tailor-platform/styled-systems",
   }),
 );
```

storybookもひとつのアプリケーションとして扱うようにすることで、design-systemsがpackageとして正しい状態になっていることをチェックしたいのが理由です。

Reply:
朝会で聞いていた現象が発生して色々試していたのですが、原因判明しました。
```
+    importMap: "@tailor-platform/design-system",
+    outdir: "@tailor-platform/styled-systems",
```
ここでtypoしているのが原因でした。正しくは↓でした。
```
+    importMap: "@tailor-platform/design-systems",
+    outdir: "@tailor-platform/styled-system",
```